content/en/blog/news/The_Role_of_CloudWeGo_in_Modern_Cloud_Native_Applications/index.md

```diff
@@ -0,0 +1,159 @@
+---
+date: 2023-11-28
+title: "The Role of CloudWeGo in Modern Cloud-Native Applications"
+projects: ["CloudWeGo"]
+linkTitle: "The Role of CloudWeGo in Modern Cloud-Native Applications"
+keywords: ["CloudWeGo", "middleware", "Kitex", "microservice framework", "ByteDance Open Source", "open source", "cloud native", "open source", "kubernetes", "gRPC", "microservices"]
+description: "Explore CloudWeGo, ByteDance's robust set of microservices middleware, in this detailed look at its role in the modernization of IT infrastructure. Discover CloudWeGo's versatility in facilitating cloud-native applications, its unique advantages, and its impact across various industries."
+author:  <a href="https://github.com/yy2so" target="_blank">Yacine Si Tayeb</a>, <a href="https://github.com/GuangmingLuo" target="_blank">Guangming Luo</a>
+---
+
+
+![Image](img/blog/The_Role_of_CloudWeGo_in_Modern_Cloud_Native_Applications/1.png)
```

Comment:
It's not working. Please add a / in front of the path of all the images.
e.g. /img/blog/The_Role_of_CloudWeGo_in_Modern_Cloud_Native_Applications/1.png

Reply:
Done, please check @GuangmingLuo